composer.json

```diff
@@ -3,17 +3,17 @@
     "description": "Redirects and 404 tracking for Kirby",
     "license": "MIT",
     "type": "kirby-plugin",
-    "version": "4.4.2",
+    "version": "4.5.0",
     "authors": [
         {
             "name": "Nico Hoffmann",
             "email": "on-github@nhoffmann.com"
         }
     ],
     "require": {
-        "php": ">=8.0.0 <8.3.0",
+        "php": "^8.0",
```

Comment:
I don't think this is a good idea - we don't know what breaking changes PHP 8.4 could bring and if this version works with it. So if PHP 8.3 support should be allowed (and you tested it that it works), I'd change this to `<8.4.0`.